src/Raven.Server/Documents/PeriodicExport/PeriodicExportStatus.cs

```diff
@@ -8,5 +8,7 @@ public class PeriodicExportStatus
         public DateTime LastFullExportAt { get; set; }
 
         public long? LastDocsEtag { get; set; }
+
+        public string LastFullExportFolder { get; set; }
```

Comment:
You aren't saving this value. On restart, we'll lose this value.
See `PeriodicExportRunner.WriteStatus`